vm/class_test.go

```diff
@@ -170,29 +170,61 @@ func TestCustomClassConstructor(t *testing.T) {
 	v.checkSP(t, 0, 1)
 }
 
-func TestDefClassMethod(t *testing.T) {
+func TestSendMethod(t *testing.T) {
 	tests := []struct {
 		input    string
-		expected int
+		expected interface{}
 	}{
 		{`
+		def foo
+		  10
+		end
+
+		send(:foo)
+		`, 10},
+		{`
 		class Foo
-		  def self.bar
+		  def bar
 		    10
 		  end
 		end
 
-		Foo.bar
+		Foo.new.send(:bar)
 		`, 10},
 		{`
-		module Foo
+		class Foo
 		  def self.bar
 		    10
 		  end
 		end
 
-		Foo.bar
+		Foo.send(:bar)
 		`, 10},
+		{`
+		def foo(x)
+		  10 + x
+		end
+
+		send(:foo, 5)
+		`, 15},
+		{`
+		class Foo
+		  def bar(x)
+		    10 + x
+		  end
+		end
+
+		Foo.new.send(:bar, 5)
+		`, 15},
+		{`
+		class Foo
+		  def self.bar(x)
+		    10 + x
+		  end
+		end
+
+		Foo.send(:bar, 5)
+		`, 15},
 	}
```

Comment:
Should there be module tests as well?

Reply:
Class and module are basic the same internally, so I don't think it's necessary